docs/src/pages/presets/writing-presets/index.md

```diff
@@ -3,6 +3,8 @@ id: 'writing-presets'
 title: 'Writing Presets'
 ---
 
+> migration guide: This page document the method to configure storybook introduced recently in 5.3.0, consult the [migration guide](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md) if you want to migrate to this format of configuring storybook.
+
 [Storybook presets](../introduction/) are grouped collections of `babel`, `webpack`, and `addons` configurations that support specific use cases in Storybook, such as typescript or MDX support.
```

Comment:
"Addons" needs to be updated

Reply:
What do you mean by this?

Reply:
What do I need to change it to?

Reply:
This page documents `entries` and `managerEntries`, but does not document `addons` or provide examples (addons is empty `[]`) up and down the page. we should document it for completeness, esp. since this changed.

Reply:
ok, thanks for he explanation, I'll add it

Reply:
How does this look?